frontend/Dockerfile

```diff
@@ -142,6 +142,8 @@ COPY --from=wasm_files . /usr/local/apache2/htdocs/assets/webnode/pkg/
 # Copy circuit blobs for devnet and mainnet from local directory
 COPY circuit-blobs/3.0.1devnet /usr/local/apache2/htdocs/assets/webnode/circuit-blobs/3.0.1devnet
 COPY circuit-blobs/3.0.0mainnet /usr/local/apache2/htdocs/assets/webnode/circuit-blobs/3.0.0mainnet
+COPY circuit-blobs/berkeley-devnet /usr/local/apache2/htdocs/assets/webnode/circuit-blobs/berkeley-devnet
+COPY circuit-blobs/berkeley_rc1 /usr/local/apache2/htdocs/assets/webnode/circuit-blobs/berkeley_rc1
```

Comment:
Only 3.0.0mainnet and berkeley-devnet should be required, as in the [Makefile](https://github.com/o1-labs/mina-rust/blob/develop/Makefile#L218-L219). We should avoid adding more to the Makefile as it does not come for free (increase the size of the Dockerfile). Can you update accordingly please?